src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -16,6 +16,7 @@ namespace System.Diagnostics
     ///     used in conjunction with the <see cref='System.Diagnostics.Process'/>
     ///     component.
     /// </devdoc>
+    [DebuggerDisplay("FileName={FileName}, Arguments={BuildArguments()}, WorkingDirectory={WorkingDirectory}")]
```

Comment:
How did you decide on these three specific members being the most important information from the PSI to surface?

Reply:
In #27251 @kichalla asked for escaped arguments, based on my debugging experience of `Process.Start` failures I thought it would be nice to extend it with `FileName` and `WorkingDirectory` to know what exactly and where we are trying to start.

Reply:
Filename and arguments make sense:, if for no other reason than the API treats those as special with a ctor that lets you specify them. I'm less sold on working directory. For one, I don't see why that in particular is more or less important than, e.g., whether stdin/out/err are redirected, or environment variables. But second, it's also potentially misleading, since an empty WorkingDirectory has a special meaning in the implementation but the tooltip won't (and shouldn't) reflect that.

Reply:
The difference is that whether stdin/out/err are redirected typically does not affect the success of `Process.Start`, while having invalid `WorkingDirectory` does. I've personally hit such an issue few times and I believe that in the context of what I am trying to start it's important to know: what (FileName), where (WorkingDirectory) and how (Arguments). Since `WorkingDirectory` is not as frequently set as `Arguments`, it's the last thing in the tooltip.

Reply:
> does not affect the success of Process.Start

Sure it does.